src/ecdsa/robust_ecdsa/sign.rs

```diff
@@ -17,13 +17,16 @@ use crate::{
 };
 type C = Secp256K1Sha256;
 
+/// Depending on whether the current participant is a coordinator or not,
+/// runs the signature protocol as either a participant or a coordinator.
 pub fn sign(
     participants: &[Participant],
+    coordinator: Participant,
```

Comment:
Passing the coordinator around feels redundant if all we end up doing is comparing it with "me". Could we simplify this to?
```suggestion
    is_coordinator: bool,
```